### ISSUE TITLE
Qiime metadata shouldn't have bioinformatics terms

### ISSUE THREAD

**Issue by ksilnoaa** [aomlomics/FAIRe2QIIME#1]
When creating the metadata file that will be used in Tourmaline, we probably don't want any of the Bioinformatics terms, as those will be set in the Tourmaline run itself and we don't want it to be confusing. In Tourmaline V2, the metadata file is optional as it is just if you want plots to include sample metadata (eg, ordering by metadata in the taxa barplot).

**Comment by lukenoaa** [comment]
Agree. Do you think we can just skip the whole "Bioinformatics" seciton?
